README.rst

```diff
@@ -22,43 +23,102 @@ Query Parameters:
+
+Finally, you can start the server with::
 
-    pip install invoke
-    invoke requirements
-    export AWS_ACCESS_KEY_ID="..."
-    export AWS_SECRET_ACCESS_KEY="..."
-    export AUTOAPI_BUCKET="..."
     invoke serve
 
-    curl http://localhost:5000
-    curl http://localhost:5000/sample
-    curl http://localhost:5000/sample/1
-    curl http://localhost:5000/sample?LastName=SMITH
+Now try visiting http://localhost:5000 in your browser, or poke at
+the following URLs from the command-line::
 
-Via docker
-----------
+    curl http://localhost:5000/my_sample_data
+    curl http://localhost:5000/my_sample_data/1
+    curl http://localhost:5000/my_sample_data?color=pink
+
+More details can be found in `instructions.md <https://github.com/18F/autoapi/blob/master/instructions.md>`_.
+
+
+Deployment via docker
+---------------------
 
 If you have `Docker <http://docker.io>`_ installed, you can run from a Docker
 container with no further setup::
 
     docker run \
       -p 5000:5000 \
-      -e "AUTOAPI_NAME=$AUTOAPI_NAME" \
-      -e "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID" \
-      -e "AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY" \
-      -e "AUTOAPI_BUCKET=$AUTOAPI_BUCKET" \
       -v `pwd`/data_sources:/data_sources \
       --rm -it 18fgsa/autoapi
 
+Any environment variables can be set via the ``-e NAME=VALUE`` option.
+
+Note that this sets up your autoapi instance by directly
+pulling autoapi from its
+`Docker Hub image <https://hub.docker.com/r/18fgsa/autoapi/>`_; it doesn't
+even require you to clone autoapi's git repository. However, as a
+result you won't be able to develop autoapi itself.
+
+
+Development via docker
+----------------------
+
+If you'd like to use Docker for developing autoapi, just clone its
+repository, build the docker image and start the server::
+
+    docker-compose build
+    docker-compose up
+
+Now you can visit your server at http://localhost:5000.
+
+If you want to set any environment variables, you can do so by creating
+a ``.env`` file in the root directory of the repository, where each line
+consists of a ``NAME=VALUE`` pair.
+
+If any dependencies change, such as those listed in ``requirements.txt``
+or ``package.json``, just re-run ``docker-compose build``.
+
+For more information on using Docker for development, see the
+`18F Docker Guide <https://pages.18f.gov/dev-environment-standardization/virtualization/docker/>`_.
+
+
 Database Configuration
 ----------------------
 
-By default, autoapi uses a local SQLite database. To specify a different database URI, set the `DATABASE_URL` environment variable. For use with Cloud Foundry, simply create and bind an RDS service; this will automatically configure the environment.
+By default, autoapi uses a local SQLite database. To specify a different database URI, set the ``DATABASE_URL`` environment variable. For use with Cloud Foundry, simply create and bind an RDS service; this will automatically configure the environment.
```

Comment:
It looks like restructured text requires double-backticks rather than single-backticks for monospaced typesetting.